packages/firebase_remote_config/firebase_remote_config/example/lib/main.dart

```diff
@@ -33,21 +34,38 @@ class WelcomeWidget extends AnimatedWidget {
       appBar: AppBar(
         title: const Text('Remote Config Example'),
       ),
-      body: Center(child: Text('Welcome ${remoteConfig.getString('welcome')}')),
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Text('Welcome ${remoteConfig.getString('welcome')}'),
+            SizedBox(
+              height: 20,
+            ),
+            Text('(${remoteConfig.getValue('welcome').source})'),
+            Text('(${remoteConfig.lastFetchTime})'),
+            Text('(${remoteConfig.lastFetchStatus})'),
+          ],
+        ),
+      ),
       floatingActionButton: FloatingActionButton(
           child: const Icon(Icons.refresh),
           onPressed: () async {
             try {
-              // Using default duration to force fetching from remote server.
-              await remoteConfig.fetch(expiration: const Duration(seconds: 0));
-              await remoteConfig.activateFetched();
-            } on FetchThrottledException catch (exception) {
-              // Fetch throttled.
+              // Using zero duration to force fetching from remote server.
+              await remoteConfig.setConfigSettings(RemoteConfigSettings(
```

Comment:
Does it mean that after one manual refresh, the configs will no-longer refresh every hour (if that's what `minimumFetchInterval` is about)?

Reply:
Configs are only updated as a result of a `fetch` call. If the `minimumFetchInterval` has been exceeded since the last `fetch` then the `fetch` will go to the server for the latest values otherwise cached values will be used.

So in this case with a zero `minimumFetchInterval` each fetch will go to the server. But this not recommended since config values don't change that often. After setting the `minimumFetchInterval` to zero to force a fetch it would be recommended to set it back to something more reasonable like 1 hour.